src/Stores/rootReducer.js

```diff
@@ -3,11 +3,13 @@ import { combineReducers } from 'redux';
 import PoliticiansReducer from './Politicians/reducer';
 import PromisesReducer from './Promises/reducer';
 import SubjectsReducer from './Subjects/reducer';
+import CommonReducer from './Common/reducer';
```

Comment:
whats CommonReducer? ErrorReducer a better name?

Reply:
or snackbar reducer?

Reply:
My thought process is that Shared stuff should go here.

Although I might be over designing the reducer right now 🤔

I'm not too thrilled about `snackBarReducer` though, since the purpose of this reducer is to send a message to the user. Be it success or error

Reply:
feedbackReducer?